requirements.txt

```diff
@@ -8,6 +8,7 @@ ashes==19.2.0
 atomicwrites==1.3.0       # via pytest
 attrs==19.1.0             # via glom, pytest
 boltons==19.1.0
+chameleon==3.6
```

Comment:
Please add this to requirements.in (as just chameleon, with no version) so that it gets pulled in next time I pin deps with pip-tools. :)